### PR TITLE
Fix dec2oct

### LINKED ISSUE #147: Fix "dec2oct" method
There is currently a method called `dec2oct` that lives in `/lib/dec2oct.js`.
It is incomplete and needs to be fixed!

### Acceptance Criteria:
1. Running `yarn test dec2oct` results in tests passing.
2. You have written a skeleton method for someone else to work on.
3. You have written tests surrounding your skeleton method.
4. Running `yarn lint` does not print any errors to the console!

### Please include the skeleton of a new method + an accompanying test for someone else to work on at the time of creating a pull request! A pull request will most likely be denied if it does not contain a skeleton method for someone else to work on! For more information, please read the [Contributing Guide](https://github.com/BlakeGuilloud/ganon/blob/master/CONTRIBUTING.md).

Thank you so much for your contribution!

### PR DESCRIPTION
**Pull request by v-chmykov**
- [] Running `yarn lint` does not trigger any linter errors
- [] The test of the method you have fixed is passing
- [] You have written a new skeleton method for someone else to work on!
- [] You have written tests to accompany your skeleton method

Closes #147

### CODE REVIEW

File: lib/dec2oct.js [hunk 1]
```diff
@@ -1,7 +1,21 @@
...
+    octal = 0;
+  } else {
+    while(number > 0) {
+      remainder = number % 8;
+      octal = remainder + octal;
+      number = Math.floor(number / 8);
+    }
+  }
+
+  return octal;
```
Comment (ktilcu) [inline lib/dec2oct.js]: Great work on this. You may wanna have a look at the `toString` method. It has the ability to convert numbers to different bases. `parseInt` can parse a number from a string in many bases too.

**Comment by ktilcu** [comment]
If you fix the merge conflict I will merge the PR.

File: test/isStringUppercase.test.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
+const { isStringUppercase } = require("../lib");
+
+test("isStringUppercase returns false for single char 'c'", () => {
```
Comment (BlakeGuilloud) [inline test/isStringUppercase.test.js]: Hey @v-chmykov ! Awesome work on this. We are trying to move towards wrapping all of our tests into a single "describe" block. See other .test.js files for examples! If you resolve this issue, I will happily merge this in! Thanks!!

**Comment by BlakeGuilloud** [comment]
Hey @v-chmykov - I am seeing some linter errors in our Travis CI build. Could you please fix those so we can merge this in? Thanks!

```
/home/travis/build/BlakeGuilloud/ganon/test/isStringUppercase.test.js
 4:1 error Expected indentation of 2 spaces but found 1 tab indent
 5:2 error Mixed spaces and tabs no-mixed-spaces-and-tabs
 6:1 error Expected indentation of 2 spaces but found 1 tab indent
 8:1 error Expected indentation of 2 spaces but found 1 tab indent
 9:2 error Mixed spaces and tabs no-mixed-spaces-and-tabs
 10:1 error Expected indentation of 2 spaces but found 1 tab indent
 12:1 error Expected indentation of 2 spaces but found 1 tab indent
 13:2 error Mixed spaces and tabs no-mixed-spaces-and-tabs
 14:1 error Expected indentation of 2 spaces but found 1 tab indent
 16:1 error Expected indentation of 2 spaces but found 1 tab indent
 17:2 error Mixed spaces and tabs no-mixed-spaces-and-tabs
 18:1 error Expected indentation of 2 spaces but found 1 tab indent
 20:1 error Expected indentation of 2 spaces but found 1 tab indent
 21:2 error Mixed spaces and tabs no-mixed-spaces-and-tabs
 22:1 error Expected indentation of 2 spaces but found 1 tab indent
```

**Comment by ktilcu** [comment]
@v-chmykov I know you've worked hard on this. I really wanna se it make it into the codebase. Can you clean up the linter errors? We are so close to the finish line on this one.

**Pull request merged**